webodm.sh

```diff
@@ -576,13 +572,11 @@ elif [[ $1 = "stop" ]]; then
 
 	command="$docker_compose -f docker-compose.yml"
 
-	if [ "${GPU_NVIDIA}" = true ]; then
-		command+=" -f docker-compose.nodeodm.gpu.nvidia.yml"
-	elif [ "${GPU_INTEL}" = true ]; then
-		command+=" -f docker-compose.nodeodm.gpu.intel.yml"
-	else
-		command+=" -f docker-compose.nodeodm.yml"
-	fi
+
+	command+=" -f docker-compose.nodeodm.gpu.nvidia.yml"
+	command+=" -f docker-compose.nodeodm.gpu.intel.yml"
+	command+=" -f docker-compose.nodeodm.yml"
```

Comment:
I'm not sure I understand the change here.

Reply:
The shutdown function now has to be entered after the `--gpu` to shut down the gpu node. But it's just the startup parameter in the manual. There may be users who get unexpected results. I would like to be able to make sure they shut down.